Sources/Honeycomb/HoneycombNavigationSpanInstrumentation.swift

```diff
@@ -0,0 +1,115 @@
+import OpenTelemetryApi
+import OpenTelemetrySdk
+import SwiftUI
+
+private let navigationInstrumentationName = "@honeycombio/instrumentation-navigation"
+private let navigationSpanName = "Navigation"
+private let unencodablePath = "<unencodable path>"
+
+func getTracer() -> Tracer {
+    return OpenTelemetry.instance.tracerProvider.get(
+        instrumentationName: navigationInstrumentationName,
+        instrumentationVersion: honeycombLibraryVersion
+    )
+}
+
+internal class HoneycombNavigationProcessor {
+    static let singleton = HoneycombNavigationProcessor()
```

Comment:
fyi, if this were a public API, I would recommend naming it either `instance` or `shared`, as those are more idiomatic, but for an internal instance, I don't think it matters.

Reply:
I'll switch to `shared`